changelog/2.079.0.dd

```diff
@@ -795,7 +795,7 @@ For example, on a Posix platform you can simply run:
 
 $(P
 $(CONSOLE
-sed "s/delete \(.*\);/__delete(\1);/" -i **/*.d
+`sed "s/delete \(.*\);/__delete(\1);/" -i **/*.d`
```

Comment:
Perhaps we should replace all uses of `$(CONSOLE ...)` with something along the lines of:

    ```console
    ```

Reply:
Again, the reason I didn't make a change like this (and I was tempted to) is to minimize the amount of changes.